bin/mod/ent-prj

```diff
@@ -62,6 +62,7 @@ RUN() {
       prj_generate_custom_resource "$@"
       ;;
     "deploy") #H: generates the CR and deploys it to the current profile
+      kube.utils.is_api_server_reachable || _FATAL -s "Unable to connect to the Entando application"
```

Comment:
Same is needed for install, no?

Reply:
`ent prj install` and `ent prj uninstall` use the `ecr-prepare-action` so they are covered